web/src/beta/components/ActionItem/index.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import ActionItem from ".";
+
+export default {
+  component: ActionItem,
+} as Meta;
+
+type Story = StoryObj<typeof ActionItem>;
+
+export const Default: Story = {
+  render: () => <ActionItem title={"New Page"} icon={"square"} />,
```

Comment:
same as [this comment](https://github.com/reearth/reearth/pull/495/files#r1226070032)